lib/background_worker.rb

```diff
@@ -30,6 +30,12 @@ def self.verify_active_connections!
     end
   end
 
+  def self.release_connections!
+    if defined?(ActiveRecord) && ActiveRecord::VERSION::MAJOR == 4
+      ActiveRecord::Base.clear_all_connections!
```

Comment:
Yeah I just went with the specific 4 to fix the problem, I'd say when we upgrade to 5 we see what happens. If it breaks in the same way we change the condition.